experimental/InvariantTheory/invariant_rings.jl

```diff
@@ -385,7 +385,7 @@ end
 
 function _molien_series_via_singular(S::PolyRing, IR::InvRing{T}) where {T <: Union{Nemo.GaloisField, Nemo.GaloisFmpzField}}
   @assert !ismodular(IR)
-  mol = molien_series_via_singular(IR)
+  mol = __molien_series_via_singular(IR)
 
   # TODO: Write a conversion from Singular number fields to Nemo ones
```

Comment:
This should probably go under the `oscar_ring` function?

Reply:
This is something from @joschmitt's previous PR.

Reply:
Yes, it would be nice to have `oscar_ring` for converting a Singular number field to Nemo.